agent/common.sh

```diff
@@ -11,8 +11,14 @@ export CLUSTER_HOST_PREFIX_V4="24"
 export EXTERNAL_SUBNET_V6="fd2e:6f44:5dd8:c956::/64"
 
 if [ -n "$MIRROR_IMAGES" ]; then
-    # We're going to be using a locally modified release image
-    export OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE="${LOCAL_REGISTRY_DNS_NAME}:${LOCAL_REGISTRY_PORT}/localimages/local-release-image:${OPENSHIFT_RELEASE_TAG}"
+
+    export MIRROR_IMAGE_URL_SUFFIX="localimages/local-release-image"
+
+    if [ "${OC_MIRROR}" == true ]; then
+        export MIRROR_IMAGE_URL_SUFFIX="openshift/release-images"
+    fi
+
+    export OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE="${LOCAL_REGISTRY_DNS_NAME}:${LOCAL_REGISTRY_PORT}/${MIRROR_IMAGE_URL_SUFFIX}:${OPENSHIFT_RELEASE_TAG}"
```

Comment:
This second override here it's not clear. I would consider the previous one just a temporary workaround until it will possible to specify a given version/pullspec (otherwise this won't work on CI, especially for disconnected builds), but wouldn't see the need to repeat it here

Reply:
> This second override here it's not clear. I would consider the previous one just a temporary workaround until it will possible to specify a given version/pullspec (otherwise this won't work on CI, especially for disconnected builds), but wouldn't see the need to repeat it here

Yeah, may need to be able to clean this up so OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE is just set for OC_MIRROR and we use the one defined in common.sh in all other cases, although I'm not sure about the concern with CI as its set to the same value.